rmgpy/cantherm/qchem.py

```diff
@@ -115,19 +115,30 @@ def loadGeometry(self):
         """
         atom = []; coord = []; number = []; 
 
+        #First check that the Qchem job file (not necessarily a geometry optimization)
+        #has successfully completed, if not an error is thrown
+        completedjob = 0;
         f = open(self.path, 'r')
         line = f.readline()
         while line != '':
-            if 'Final energy is' in line:
-                print 'found a sucessfully completed Qchem Geometry Optimization Job'
+            if 'Total job time:' in line:
+                print('Found a sucessfully completed Qchem Job')
+                completedjob = 1
                 line = f.readline()
                 atom = []; coord = []
                 break
             line = f.readline()
-        found = 0           
+        f.close()
+        if completedjob != 1:
+            raise Exception('Could not find a successfully completed Qchem job')
```

Comment:
Could you add `self.path` to the output message to make it more useful in case there are several QChem jobs in a single run?

Also, please use `InputError` instead of `Exception` (you'll have to import it: `from rmgpy.exceptions import InputError`). There are three other `Exception`s in this file, would you mind changing them as well?